packages/cli/src/commands/addons/upgrade.ts

```diff
@@ -0,0 +1,151 @@
+import color from '@heroku-cli/color'
+import {Command, flags} from '@heroku-cli/command'
+import {Args, ux} from '@oclif/core'
+import {formatPriceText} from '../../lib/addons/util'
+import {addonResolver} from '../../lib/addons/resolve'
+import type {AddOn, Plan} from '@heroku-cli/schema'
+import {HTTP} from 'http-call'
+import {HerokuAPIError} from '@heroku-cli/command/lib/api-client'
+
+export default class Upgrade extends Command {
+  static aliases = ['addons:downgrade'];
+  static topic = 'addons'
+  static description = 'change add-on plan'
+  static help = 'See available plans with `heroku addons:plans SERVICE`.\n\nNote that `heroku addons:upgrade` and `heroku addons:downgrade` are the same.\nEither one can be used to change an add-on plan up or down.\n\nhttps://devcenter.heroku.com/articles/managing-add-ons'
+  static examples = ['Upgrade an add-on by service name:\n$ heroku addons:upgrade heroku-redis:premium-2\n\nUpgrade a specific add-on:\n$ heroku addons:upgrade swimming-briskly-123 heroku-redis:premium-2']
+  static flags = {
+    app: flags.app(),
+  }
+
+  static args = {
+    addon: Args.string({required: true}),
+    plan: Args.string(),
+  }
+
+  private parsed = this.parse(Upgrade)
+
+  public async run(): Promise<void> {
+    const ctx = await this.parsed
+    const {flags: {app}, args} = ctx
+    // called with just one argument in the form of `heroku addons:upgrade heroku-redis:hobby`
```

Comment:
This comment is also on line 88.. was this an unintentional copypaste?

Reply:
Removed.